graphium/trainer/predictor.py

```diff
@@ -161,6 +163,10 @@ def __init__(
         self.validation_step_outputs = []
         self.test_step_outputs = []
         self.epoch_start_time = None
+        
+        # Decide whether to log every step or once at the end 
+        # of the epoch.
+        self.metrics_every_n_steps = metrics_every_n_steps
```

Comment:
Is this related to the reporting of metrics during training? Reporting metrics every step might have some RAM issue? I wonder if we could set the default to be false?